src/stan/lang/grammars/semantic_actions_def.cpp

```diff
@@ -1625,8 +1625,11 @@ namespace stan {
     void deprecated_integrate_ode::operator()(std::ostream& error_msgs)
       const {
       error_msgs << "Warning: the integrate_ode() function is deprecated"
-             << " in the Stan language; use integrate_ode_rk45() [non-stiff]"
-             << " or integrate_ode_bdf() [stiff] instead."
+             << " in the Stan language; use the following functions"
+             << " instead."
+             << "  integrate_ode_rk45() [non-stiff]"
+             << "  integrate_ode_adams() [non-stiff]"
```

Comment:
This is where it'd be nice to have a suggestion about which to use when.

Reply:
Updated msg
```c++
       error_msgs << "Warning: the integrate_ode() function is deprecated"
             << " in the Stan language; use the following functions"
             << " instead.\n"
             << " integrate_ode_rk45() [explicit, order 5, for non-stiff problems]\n"
             << " integrate_ode_adams() [implicit, up to order 12, for non-stiff problems]\n"
             << " integrate_ode_bdf() [implicit, up to order 5, for stiff problems]."
             << std::endl;
```

Reply:
Thanks---that's great.  I can use that in the manual then, too.